internal_transfers/actions/src/parse.ts

```diff
@@ -91,3 +97,27 @@ export function tryParseERC20Transfer(log: Log): TransferEvent | null {
     amount: BigInt(value),
   };
 }
+
+export function tryParseWethAction(log: Log): TransferEvent | null {
+  if (log.address !== WETH_TOKEN_ADDRESS) return null; // Not emitted by WETH Contract!
+  const eventLog = wethInterface.parseLog(log);
+  if (
+    eventLog === null ||
+    !["Deposit", "Withdrawal"].includes(eventLog.name) ||
+    eventLog.args.address
```

Comment:
Oh I think I see this... might have been a mistake/typo -- I was trying to fined the address field in the event Log (but it does not seem to exist).
```suggestion
```